test/system/helpers.bash

```diff
@@ -278,6 +278,24 @@ function wait_for_ready {
     wait_for_output 'READY' "$@"
 }
 
+###################
+#  wait_for_port  #  Returns once port is available on host
+###################
+function wait_for_port() {
+    local host=$1                      # Probably "localhost"
+    local port=$2                      # Numeric port
+    local timeout=${3:-5}              # Optional; default to 5 seconds
+
+    # Wait
+    while [ $timeout -gt 0 ]; do
+        { exec 3<> /dev/tcp/$host/$port; } &>/dev/null && return
+        sleep 1
+        _timeout=$(( $_timeout - 1 ))
```

Comment:
The variable name is misspelled.
```suggestion
        timeout=$(( $timeout - 1 ))
```

Reply:
@edsantiago I copy-pasted that part, was there a source for this code that also needs a typo-fixed?

Reply:
https://github.com/containers/podman/pull/10984

Reply:
Oh goodie...want me to fix this one to use the same `_timeout` style?

Reply:
nevermind, an unrelated test failed anyway.  I fixed this to match the `_timeout` style.